src/tests/helpers/api.ts

```diff
@@ -69,6 +69,7 @@ export class API {
     height = width,
     isDeleted = false,
     groupIds = [],
+    colorRanges,
```

Comment:
I believe this `createElement` method is for `ExcalidrawElement`s in general, and `colorRanges` is for `ExcalidrawTextElement`s (and/or custom `ExcalidrawElement`s which generalize `ExcalidrawTextElement`). So, you may want to omit this line and handle `colorRanges` from the `...rest` parameters instead.